rpp30-boc-meltdown/biteshare
# More functionality to current session summary tab

## Description

Added the 'swipe to remove guest' functionality, the 'click to view ordered items' functionality and the overall status indicator.
I also created a `MenuItemCard` component that ideally can be shared with `current session - bills' tab. 

## How to test

Pull this branch and run the build for ios, the current session tab should look like this:
![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/43324065/147302043-22d1a225-a341-4343-a8e8-cec4f65615ee.gif)

And if everyone is ready, the status indicator and split bill options will have a style change:
![Screen Shot 2021-12-23 at 1 01 00 PM](https://user-images.githubusercontent.com/43324065/147301446-9f3a3dcf-1040-44b4-8fd3-5b72e79df3f6.png)


## Notes

I currently hardcoded 'Greg' to be ready and that's why clicking 'allow' for him will jump to the 'ready' status.

There are so many conditions on this page so apologize if it's confusing. I'm sometimes confused as well lol. Please do give suggestions or advice if there are better ways!

## Issue tracking

https://trello.com/c/TOgwDNp5
https://trello.com/c/sdFzQcSU
https://trello.com/c/TNK2DOF7
